### DISCUSSION TITLE
Can I have a page with the root "/" endpoint that isn't named "index.vue"?

### DISCUSSION THREAD

**Discussion by hotsaucejake** [posva/unplugin-vue-router#123] [Questions and Help] [answered]
I simply want a landing page to be name something other than index.vue because I'm trying reserve the "/index/" prefix for routes under an index folder.  In my case, "/index/" will be authenticated routes.

Something like "home.vue" will be accessed at "/"

And "index.vue" will use the index folder and all of it's subpages for "/index" or "/index/subpage"

**Comment by posva** [comment, marked as answer]
You can override the `path` with the route block or `definePage({ path: '...' })`
